go-dlv.el

```diff
@@ -108,7 +108,8 @@
     output))
 
 (define-obsolete-variable-alias 'go-dlv-command-name
-  'gud-dlv-command-name)
+  'gud-dlv-command-name
```

Comment:
can you please explain this change in detail?

Reply:
Emacs 28 change the signature of `define-obselete-variable-alias` to
```
define-obsolete-variable-alias is a macro defined in byte-run.el.

Signature
(define-obsolete-variable-alias OBSOLETE-NAME CURRENT-NAME WHEN &optional DOCSTRING)
```

Reply:
Please put this into the commit message and change '28' to 'emacs 28' in the commit title.